typescript/sdk/src/providers/transactions/submitter/TxSubmitter.ts

```diff
@@ -0,0 +1,30 @@
+import { ChainName } from '../../../types.js';
+import { MultiProvider } from '../../MultiProvider.js';
+import {
+  TypedTransaction,
+  TypedTransactionReceipt,
+} from '../../ProviderType.js';
+
+import { TxSubmitterType } from './TxSubmitterTypes.js';
+
+export interface TxSubmitterInterface<
+  TX extends TypedTransaction,
+  TR extends TypedTransactionReceipt,
+> {
+  /**
+   * Defines the type of tx submitter.
+   */
+  txSubmitterType: TxSubmitterType;
+  multiProvider: MultiProvider;
```

Comment:
as multiprovider is EVM specific, should there be an EvmTxSubmitterInterface that contains this instead??

i.e. if we build upon above comment
```ts
export interface EvmTxSubmitterInterface
  extends TxSubmitterInterface<ProtocolType.Ethereum> {
  multiProvider: MultiProvider;
}
```

each submitter then implements EvmTxSubmitterInterface

Reply:
This is a great callout– keeping MP here was an oversight on my part. Moving this out in the next commit